xaitk_saliency/utils/bin/sal_on_coco_dets.py

```diff
@@ -59,6 +65,8 @@ def sal_on_coco_dets(
     :param config_file: Config file specifying the ``DetectImageObjects``,
         ``PerturbImage``, and ``GenerateDetectorProposalSaliency``
         implementations to use.
+    :param fill: RGB values for fill to use when occluding three-channel images. If your annotations
+        include one-channel images, this will fail.
     :param overlay_image: Overlay saliency maps on images with bounding boxes.
         RGB images are converted to grayscaleDefault is to output write out
         saliency maps by themselves.
```

Comment:
Could you also check this param description- seems like we're missing a period maybe?